.github/workflows/1_2_c_promote_patch_to_stable.yml

```diff
@@ -0,0 +1,59 @@
+name: Bump versions for extension only
+
+# This workflow covers both '1_check_for_updates' and '2_bump_versions' workflows, therefore it is another variant of the same, but specifically for extension only, without any Prisma CLI updates that would need to be handled.
+# This is used to manually promote the patch branch to stable and release a new stable version.
+
+on:
+    workflow_dispatch: 
+
+env:
+  ENVIRONMENT: ${{ secrets.ENVIRONMENT }}
+
+jobs:
+  bump:
+    runs-on: ubuntu-latest
+    steps: 
+      - uses: actions/checkout@v2
+      - name: Use Node.js
+        uses: actions/setup-node@v1
+        with:
+          node-version: '12.x'
+      - name: Install Dependencies
+        run: npm install
+      - name: checkout master if necessary
+        run: git checkout master
```

Comment:
This can probably just be removed as it is default of `checkout`?

Reply:
No, because this workflow will run on a patch branch only. Since the versions in the master branch have to updated, this checkout is necessary.